### DISCUSSION TITLE
Razer Fan Hub

### DISCUSSION THREAD

**Discussion by Akyfire** [Rem0o/FanControl.Releases#952] [Ideas]
I recently purchased a Razer fan hub, would love to see it supported like the Commander Pro. Does anyone know if that would be possible?

**Comment by Rem0o** [comment]
If Razer somehow has an API documentation on how to do so, then yes. But you can bet that's not available.

**Comment by fu-raz** [comment]
It's probably a HID device like the Commander Pro, since it's connected to a USB header. So if you are a little tech savvy you could use [Free USB Analyzer](https://freeusbanalyzer.com/) to see if you can see what packages are sent to the device to change speed and read out rpm. 

I did that for my [Thermaltake Fan Control Plugin](https://github.com/fu-raz/FanControlThermaltake).

**Comment by EvanMulawski** [comment]
Now available: https://github.com/EvanMulawski/FanControl.Razer
